### PR TITLE
Jonah Park css-brochure-form Exercise

### PR DESCRIPTION
**Pull request by jonahp001**
Kapture 2023-01-25 at 21 04 51
Kapture 2023-01-25 at 21 04 51
Kapture 2023-01-25 at 21 04 51

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  margin: auto;
+}
+
+div.fieldtext {
+  margin-bottom: 3px;
+  margin-left: 37px;
+  text-align: left;
+}
+
+input::placeholder {
```
Comment (robgmerrill) [inline css-brochure-form/styles.css:30]: Move padding to `.userinput`. That way when the user starts to type in the input field it starts at the same place as the placeholder text.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Solid start! The only fix requested is regarding the padding in the input tag when the user starts to type. I've given feedback in the comments on how to fix it. Reach out in Slack if you have any questions about that!

**Comment by jonahp001** [comment]
Kapture 2023-01-26 at 14 25 51
switched the padding location! thank you!

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**